modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/ocl/loader/data/models/vo/CategoryOclVo.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.ocl.loader.data.models.vo;
```

Comment:
i think it is better to call the package with full name instead of short form `vo`. It is then clear for everyone.